jberet-core/src/main/java/org/jberet/operations/ForceStopJobOperatorImpl.java

```diff
@@ -0,0 +1,13 @@
+package org.jberet.operations;
+
+import jakarta.batch.runtime.BatchStatus;
+import org.jberet.runtime.JobExecutionImpl;
+
+public class ForceStopJobOperatorImpl extends DefaultJobOperatorImpl {
+    public void forceStop(final long executionId) {
+        final JobExecutionImpl jobExecution = getJobExecutionImpl(executionId);
+        jobExecution.setBatchStatus(BatchStatus.STOPPED);
+        jobExecution.setLastUpdatedTime(System.currentTimeMillis());
+        getJobRepository().updateJobExecution(jobExecution, false, false);
```

Comment:
The relative step executions and partition executions need to be updated here too.